product-features/liquidity-providers/how-to.md

```diff
@@ -0,0 +1,223 @@
+---
+description: >-
+  In these two guides, you'll find the required steps to provide liquidity and
+  to withdraw liquidity from ALEX DEX pools.
+---
+
+# How to add/remove liquidity
+
+When **adding liquidity**, you will deposit an equivalent value of both tokens into the pool. In return, you’ll receive LP tokens, which represent your share of that specific liquidity pool.
+
+When **removing liquidity**, you will transfer your LP tokens back to the ALEX smart contract and receive your original funds plus any fees accrued while holding those LP tokens.
```

Comment:
@XianPaz I am not sure if these clarifications should go here at the beginning or below each "Adding/Removing Liquidity" subtitle. Plz let me know your thoughts.

Reply:
Here is OK